objmodel/chapter.md

```diff
@@ -119,10 +119,10 @@ def test_read_write_field():
     assert obj.read_attr("b") == 5
 ````
 
-The test uses three things that we will have to implement in this chapter. On
-the one hand the classes ``Class`` and ``Instance`` that represent classes and
-instances of our object model, respectively. On the other hand, there are two
-special instances of class, ``OBJECT`` and ``TYPE``. ``OBJECT`` corresponds to
+The test uses three things that we have to implement. 
+The classes ``Class`` and ``Instance`` represent classes and
```

Comment:
I rewrote this to remove "on the one hand" and "on the other hand" because you start off saying there are three things, and I didn't want our readers to be distracted by the thought of somebody with three hands.